hieradata/common.yaml

```diff
@@ -750,47 +750,105 @@ grafana::dashboards::machine_suffix_metrics: ''
 grafana::dashboards::deployment_applications:
   asset-manager: {}
   business-support-api: {}
-  calculators: {}
+  calculators:
+    # Status, duration, controller missing @fields. prefix
```

Comment:
Mostly out of interest - is this a problem with the way the data is being reported? This app has a field called `controller` rather than the expected `@fields.controller`?